modules/fluid_properties/doc/content/syntax/FluidProperties/index.md

```diff
@@ -0,0 +1,39 @@
+# FluidProperties System
+
+## Overview
+
+`FluidProperties` objects define interfaces for computing thermodynamic
+properties of fluids (liquids and gases).
+
+## Interfaces
+
+Because two independent, intensive thermodynamic properties define a
+thermodynamic state, most interfaces in these objects are of the form $f(a,b)$
+where $f$ is the desired thermodynamic property and $a$ and $b$ are independent,
```

Comment:
Yes, I can improve this page while I'm here.

Reply:
I started this improvement and am getting carried away with it. I'll do a separate pull request for those changes, since it's bound to hold things up.